dirty-script.py

```
@@ -0,0 +1,60 @@
+import re
+from collections import Counter
```

Ruff (0.11.9)

`collections.Counter` imported but unused

Remove unused import: `collections.Counter`

(F401)